test/integration/default/serverspec/install_spec.rb

```diff
@@ -2,8 +2,8 @@
 set :backend, :exec
 set :path, '/sbin:/usr/sbin:/usr/local/sbin:$PATH'
 
-describe package('php5') do
-  it { should be_installed }
+describe command('php -v') do
+  its(:stdout) { should match('PHP 5.') }
```

Comment:
Could you adjust the match so that any kind of PHP version would be ok? (eg. PHP 7 would fail with the match above)

`its(:stdout) { should match('PHP') }` - should do the trick...